web/thesauruses/php/data_types.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
+  },
+  "categories": {
+    "Numerical": [
+      "boolean",
+      "integer",
+      "float"
+    ],
+    "Strings": [
+      "string"
+    ],
+    "Compound Types": [
+      "array",
+      "associative_array"
+    ]
+  },
+  "data_types": {
+    "boolean": {
+      "name": "Boolean",
+      "code": "(bool) $value;"
```

Comment:
Instead of casting, can you just have the type? (so `bool` not `(bool) $value;`)

And same for others below.